mixins/inputMessage.js

```diff
@@ -14,15 +15,17 @@ export default {
   },
 
   shouldShowError () {
-    let errors = this.props.errors || []
+    let propErrors = this.props.errors || []
+    const stateErrors = get(this, 'state.errors') || []
```

Comment:
It might be a little weird... the "es6-ing" here was likely a `s/var/let/` because it'd be the least likely to cause error (I'm pretty sure this was my bad actually).

But yeah, if you're touching the line... got nuts! :)